sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -118,6 +118,21 @@ def test_sanity_check_pls_regression():
     assert_array_almost_equal(y_loadings_sign_flip, y_weights_sign_flip)
 
 
+@pytest.mark.parametrize("Estimator", (PLSRegression, PLSCanonical, CCA))
+def test_pls_estimators_single_target_shape(Estimator):
+    # Check shape for single-target predictions on PLS estimators
```

Comment:
Let's also add it here: ``# non-regression test for https://github.com/scikit-learn/scikit-learn/issues/19352``